### PR TITLE
Feature/nex 2059/svgz support

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/NEX-2059

TODO
- [x] Unit test

### CODE REVIEW

**Review by bartlomiejmarszal — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
Tested on local machine

File: model/compile/QtiAssetCompiler/QtiItemAssetCompiler.php [hunk 1]
```diff
@@ -100,9 +100,10 @@ private function resolve(ItemMediaResolver $resolver, string $assetUrl, string $
         return new PackedAsset($type, $mediaAsset, $link);
     }
 
-    private function getReplacementName(string $link): string
+    private function getReplacementName(PackedAsset $packedAsset): string
     {
-        return hash('crc32', basename($link));
+        $link = $packedAsset->getMediaAsset()->getMediaIdentifier();
+        return $packedAsset->getMediaAsset()->getMediaSource()->getBaseName($link);
```
Comment (gabrielfs7) [inline model/compile/QtiAssetCompiler/QtiItemAssetCompiler.php:106]: Is it covered in the test? I remember this was a critical part changed before and we had some issues. If possible would be nice to have a test for this.

**Review by gabrielfs7 — COMMENTED** [review]
Can we add tests?

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]
conflict

**Review by siwane — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**